gitpod-io/website
# feature: add request changes to blog and guides

## Description
Adds a "request changes" link to the bottom of blog posts and guides that takes them directly to GitHub.

<img width="1077" alt="image" src="https://user-images.githubusercontent.com/5633704/185109102-627d29a1-4083-409a-bae9-f2a372acc224.png">

## Related Issue(s)
https://gitpod.slack.com/archives/C01KGM9ETU6/p1660296497258159
Fixes #

## How to test
Go to any blog post and click "Request changes" at the bottom of the post. You should be taken to GitHub's "edit file" for the markdown for that post. 

Do the same for any guide.

This should work in light and dark mode as well as wide and narrow screens. 


## Release Notes

```release-note
```

## Documentation



<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2586"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2586"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
